p2p/discv5/ticket.go

```diff
@@ -697,6 +697,7 @@ func globalRandRead(b []byte) {
 	val := 0
 	for n := 0; n < len(b); n++ {
 		if pos == 0 {
+			// #nosec TODO?
```

Comment:
Could you file an issue with the warning. Otherwise, this will be immediately lost.

Reply:
The rule is: **G404: Insecure random number source (rand)** ([rules here](https://github.com/securego/gosec#available-rules))

Reply:
Done #877 !